### DISCUSSION TITLE
Plans for Adding Drag-and-Drop Column Rearrangement in Index Table?

### DISCUSSION THREAD

**Discussion by Ipriyankrajai** [Shopify/polaris-react#12189] [Q&A]
I recently explored the Shopify store index table and noticed a feature allowing users to rearrange columns by clicking the "Edit Columns" button within the filters. This functionality significantly enhances the user experience by providing a customizable view.

![image](https://github.com/Shopify/polaris/assets/32142351/add2542f-2450-440d-8955-4710c5be5405)

Are we planning to roll out a similar feature for our index table, considering we already have the `showEditColumnsButton` prop available in the `IndexFilters` component?

![image](https://github.com/Shopify/polaris/assets/32142351/d4d64f63-2394-46f9-b80e-ee30be37b150)

**Comment by ryanolf** [comment]
I would love to know if this is possible! Anybody here?
